spyderlib/widgets/editor.py

```diff
@@ -581,10 +581,16 @@ def open_fileswitcher_dlg(self):
         """Open file list management dialog box"""
         if not self.tabs.count():
             return
+        if self.fileswitcher_dlg is not None and \
+          self.fileswitcher_dlg.is_visible:
+            self.fileswitcher_dlg.hide()
+            self.fileswitcher_dlg.is_visible = False

```

Comment:
I tried to use `visible`, believe me ;-) It just didn't work, I don't know why...